### PR TITLE
[MM-27864][MM-27865] Telemetry for Cloud Onboarding

### PR DESCRIPTION
**Pull request by devinbinnie**
#### Summary
Telemetry for Cloud Onboarding

NOTES:
- There's no telemetry for the End-User Tips and Next Steps screen, looks like it was missing from the document
- The 'Download Apps' button doesn't show up on mobile, so there's only `mac` or `windows` desktop app.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-27864
https://mattermost.atlassian.net/browse/MM-27865

### CODE REVIEW

**Review by Adovenmuehle — APPROVED** [review]
LGTM!

**Review by marianunez — APPROVED** [review]
Great stuff @devinbinnie! That was a lot of detailed events to track 👍

Non-blocking comment below on a data point missing in one of the events.

File: components/next_steps_view/next_steps_tips.tsx [hunk 1]
```diff
@@ -72,13 +77,17 @@ const getDownloadButtonString = () => {
     );
 };
 
-const openAuthPage = (page: string) => {
+const openAuthPage = (page: string, isAdmin: boolean) => {
+    trackEvent(getAnalyticsCategory(isAdmin), 'click_configure_login', {method: page});
```
[the remark this answers is not in the document]
Comment (devinbinnie) [reply components/next_steps_view/next_steps_tips.tsx:81]: The last argument to `trackEvent` has that method included, `page` is always one of `'oauth' | 'saml' | 'ldap'`

[the remark this answers is not in the document]
Comment (marianunez) [reply components/next_steps_view/next_steps_tips.tsx:81]: ahh completely missed that.🙈 Awesome!

**Review by nickmisasi — APPROVED** [review]

**Review by adamjclarkson — APPROVED** [review]

**Comment by marianunez** [comment]
@devinbinnie can we cherry-pick this to `cloud-beta`?

**Pull request merged**